context.go

```diff
@@ -792,51 +906,21 @@ func (c *Context) parseOne(rootDir, filename string, reader io.Reader,
 	}
 	file.Name = relBlueprintsFile
 
-	subdirs, subdirsPos, err := getLocalStringListFromScope(scope, "subdirs")
```

Comment:
Are these deletions the only thing required to remove subdirs support?  If so, lets leave them in and then we can figure out what we want to do with legacy subdirs support in a separate patch.  Using both file list and subdirs together shouldn't cause a problem because startParseBlueprintsFile checks if the file has already been parsed.

Reply:
done

Reply:
Hmm it turns out that there exists a symlink (not in a .git repo, but specified by a manifest) named Android.bp pointing to an Android.bp in a subdirectory, which has a nonempty value for "subdirs".

Previously the parent Android.bp was the one that was getting parsed by Blueprint, so the relative file path listed in the child is supposed to be interpreted relative to the parent. Then when this change causes Blueprint to search for Android.bp files, it finds the child Android.bp and looks for the target of the subdirs relative to the child, and fails to open the result.

Do you think that that makes it worthwhile to remove subdirs support at the same time, or whether we want to fix the Android.bp files (probably involving renaming them and updating some manifests to repoint the symlink) ?